src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -56,22 +57,31 @@ public override void Initialize(IHostContext hostContext)
 
         public ActionDefinitionData Load(IExecutionContext executionContext, string manifestFile)
         {
-            var context = CreateContext(executionContext);
+            var templateContext = CreateContext(executionContext);
             ActionDefinitionData actionDefinition = new ActionDefinitionData();
+
+            // Clean up file name real quick
+            // Instead of using Regex which can be computationally expensive, 
+            // we can just remove the # of characters from the fileName according to the length of the basePath
+            string basePath = _hostContext.GetDirectory(WellKnownDirectory.Actions);
+            var fileName = manifestFile.Remove(0, basePath.Length + 1);
```

Comment:
TODO: Add sanity check here to make sure we don't accidentally error out of the filename is smaller than the total path for some reason.